shared/actions/teams.tsx

```diff
@@ -678,13 +678,13 @@ const getChannelInfo = async (
   action: TeamsGen.GetChannelInfoPayload,
   logger: Saga.SagaLogger
 ) => {
-  const {teamname, conversationIDKey} = action.payload
+  const {teamID, conversationIDKey} = action.payload
   const results = await RPCChatTypes.localGetInboxAndUnboxUILocalRpcPromise(
     {
       identifyBehavior: RPCTypes.TLFIdentifyBehavior.chatGui,
       query: ChatConstants.makeInboxQuery([conversationIDKey]),
     },
-    Constants.teamWaitingKey(teamname)
+    Constants.teamWaitingKey(Constants.getTeamNameFromID(state, teamID) ?? '')
```

Comment:
I did a lot of `Constants.teamWaitingKey(Constants.getTeamNameFromID(state, teamID) ?? '')` with the expectation that when this refactor is over we can change `Constants.teamWaitingKey` to take a team ID instead of a team name, and to find/replace all these.

I think `null` is better because it's more accurate.

Reply:
idk, upon rereading your comment I think my actual reason is that this forces us to be aware of the fact that there may not be a team name, using the type checker.